themes/ddbasic/sass/components/node/news.scss

```diff
@@ -268,6 +268,19 @@
         top: 0;
         right: 7px;
       }
+      .field-item {
+        float: left;
```

Comment:
If the intention is to make the libraries appear as a line of text I would think that using `display: inline-block;` would be more appropriate than floats which can act quite quirky.